xml/Microsoft.Extensions.Configuration/ChainedBuilderExtensions.xml

```diff
@@ -65,7 +65,7 @@
         <ReturnType>Microsoft.Extensions.Configuration.IConfigurationBuilder</ReturnType>
       </ReturnValue>
       <Parameters>
-        <Parameter Name="configurationBuilder" Type="Microsoft.Extensions.Configuration.IConfigurationBuilder" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" RefType="this" />
+        <Parameter Name="configurationBuilder" Type="Microsoft.Extensions.Configuration.IConfigurationBuilder" RefType="this" Index="0" FrameworkAlternate="dotnet-plat-ext-3.0" />
```

Comment:
@joelmartinez can we make sure this doesn't change position from one version to the next?

Reply:
@mairaw we'll probably need to do some parameter sorting to make sure ... @mimisasouvanh, can you open a devops item on this? the functionality wouldn't be changing, just doing some sorting to maintain a consistent order

Reply:
I can file one. Just point me to the right DevOps instance. I never know. That's why filing in the repo was easier 😉

Reply:
lol ... ok ... here :) https://ceapex.visualstudio.com/Engineering/